python/tk_multi_screeningroom/screeningroom.py

```diff
@@ -179,140 +204,184 @@ def launch_submit_tool(base_url, context, source_image_seq, qt_output_path=None,
     """
 
     if not base_url:
-        raise ScreeningRoomError('A base url must be specified to launch the submit tool')
-    
+        raise ScreeningRoomError(
+            "A base url must be specified to launch the submit tool"
+        )
+
     if not source_image_seq:
-        raise ScreeningRoomError('A source image sequence must be specified to launch the submit tool')
-    
+        raise ScreeningRoomError(
+            "A source image sequence must be specified to launch the submit tool"
+        )
+
     # Generate the url for the submit tool
-    base_url = base_url.rstrip('/')
-    url = '%s/page/review_app_submit' % base_url
```

Comment:
Hound pointed out this wasn't used, so I removed it.
I'm a bit curious as to why it wasn't used but that does seem to be the case, so it seems safe to remove.

Reply:
Also the same thing in the method below as well.